src/UnitTests/ValueTransformers.cs

```diff
@@ -0,0 +1,247 @@
+﻿using Shouldly;
+using Xunit;
+
+namespace AutoMapper.UnitTests
+{
+    namespace ValueTransformers
+    {
+        public class BasicTransforming : AutoMapperSpecBase
+        {
+            public class Source
+            {
+                public string Value { get; set; }
+            }
+
+            public class Dest
+            {
+                public string Value { get; set; }
+            }
+
+            protected override MapperConfiguration Configuration { get; } = new MapperConfiguration(cfg =>
+            {
+                cfg.CreateMap<Source, Dest>();
+                cfg.ApplyTransform<string>(dest => dest + " is straight up dope");
+            });
+
+            [Fact]
+            public void Should_transform_value()
+            {
+                var source = new Source
+                {
+                    Value = "Jimmy"
+                };
+                var dest = Mapper.Map<Source, Dest>(source);
+
+                dest.Value.ShouldBe("Jimmy is straight up dope");
+            }
+        }
+
+        public class StackingTransformers : AutoMapperSpecBase
+        {
+            public class Source
+            {
+                public string Value { get; set; }
+            }
+
+            public class Dest
+            {
+                public string Value { get; set; }
+            }
+
+            protected override MapperConfiguration Configuration { get; } = new MapperConfiguration(cfg =>
+            {
+                cfg.CreateMap<Source, Dest>();
+                cfg.ApplyTransform<string>(dest => dest + " is straight up dope");
+                cfg.ApplyTransform<string>(dest => dest + "! No joke!");
```

Comment:
@jbogard @lbargaoanu I kinda like extending off the list better.  Less mass options from Intelisense and groups functionality by property.

Also can make it fluent where you add and then return list, so you can stack, and know you are stacking.